src/contexts/quiz/interface/web/quiz-routes.ts

```diff
@@ -1,17 +1,38 @@
 import { Router } from 'express';
 import { quizController } from '../controllers/quiz-controller';
 import { asyncWrapper } from '@/shared/utils/async-wrapper';
+import { QuizCreationParamsSchema } from '../../application/ports/in/create-quiz-from-material';
 
 export const quizRoutes = Router();
 
-quizRoutes.get('/:id', asyncWrapper(async (req, res) => {
-    const ids = [req.params.id];
-    const quizzes = await quizController.handleGetQuizzes(ids);
+quizRoutes.get('/:projId', asyncWrapper(async (req, res) => {
+    // Any clean way of validating that user has access to the requested project?
```

Comment:
imo call to the useCase verifies that `project.ownerId` is the authenticatedUser